python/core/symbology-ng/qgsstylev2.sip

```diff
@@ -270,6 +275,9 @@ class QgsStyleV2 : QObject
     //! gets the id from the table for the given name from the database, 0 if not found
     int getId( const QString& table, const QString& name );
 
+    //! gets the name from the table for the given id from the database, QString::null if not found
```

Comment:
Thanks for the tip!
I remember reading these pages a while ago, noting the subtle difference between `QString::isNull()` and `QString::isEmpty()`. But the discouragement of `QString::null` somehow did not catch my attention.